File: Java Sept 2023/Language Basic/prime.java
public class prime{
	public static void main(String[] args){
		int num = 70;
		int flag =0;
		if(num==1){
			System.out.println("1 is neither prime nor composite");
		}
		else{
			for(int i = 2;i<num;i++){
				if(num%i==0){
					flag = 1;	
				}
			}
		}
		if(flag==1){
			System.out.println("NOt prime.");
		}
		else{
			System.out.println("The number is prime.");
		}
	}
}

File: CPP Sept 2023/Day3/prime/prime.cpp
#include<iostream>
using namespace std;
int main(){
    int number;
    int i;
    cout<<"Enter a positive number: ";
    cin>>number;

    if(number==1){
        cout<<"one is neither prime nor composite.";
    }
    else{
        for(i = 2; i<number;i++){
            if(number%i==0){
                cout<<"The number is not prime";
                break;
            }
        }
        if(number == i){
            printf("The number is prime number.");
        }
    }

}